docs/source/conf.py

```diff
@@ -118,20 +117,28 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-#html_theme = 'default'
+html_theme = 'sphinx_rtd_theme'
 
-# pip install sphinx_bootstrap_theme
-html_theme = 'bootstrap'
-
-# Theme options are theme-specific and customize the look and feel of a theme
-# further.  For a list of options available for each theme, see the
-# documentation.
+# All sphinx_rtd_theme options. Default values commented out; uncomment to
+# change.
 html_theme_options = {
-  'bootswatch_theme': "paper",
+    'canonical_url': 'http://numba.pydata.org/numba-doc/latest/index.html',
+    # 'logo_only': False,
+    # 'display_version': True,
+    # 'prev_next_buttons_location': 'bottom',
+    'style_external_links': True,
+    # 'vcs_pageview_mode': '',
+    'style_nav_header_background': '#00A3E0',
+    # Toc options
+    # 'collapse_navigation': True,
```

Comment:
I would consider making this False. That enables digging into the TOC in the sidebar. According to the note at https://sphinx-rtd-theme.readthedocs.io/en/latest/configuring.html, if your TOC has a lot of entries it can create very large files, so you might check that first.

Reply:
Done - the gh-pages seem to take a few minutes to update, so the changes might not immediately be visible, but should be there soon.

Reply:
They seem to have updated now - load time and performance does not appear to be adversely affected.